### PR TITLE
Change translations files for breakout 2018

### PR DESCRIPTION
**Pull request by florianschmidt1994**
@thepiwo if you still have time

### CODE REVIEW

File: src/common/resources/translations/translations.de.js [hunk 1]
```diff
@@ -304,8 +304,8 @@ module.exports = {
...
-    "DESCRIPTION_4": "Das heißt die Gesamtkosten pro Team betragen 60,-€ (20€ Anmeldegebühr pro Person + 20€ Spendendeposit). Für eine erfolreiche Anmeldung muss der Gesamtbetrag bis zum 8.05.2017 überwiesen worden sein. Nutzt hierfür den <strong>angegebenen Verwendungszweck</strong> und folgende Kontoverbindung:",
+    "DESCRIPTION_3": "Da der Grundgedanke von BreakOut das Spenden an den EinDollarBrille e. V. ist, möchten wir sicherstellen, dass jedes Team diesen sozialen Gedanken unterstützt. Wir erheben deshalb 20€ Kaution pro Team. Diese überweisen wir Euch nach dem Event zurück, wenn Ihr über 100€ an Spenden generiert habt. Andernfalls gehen die 20€ in Eurem Namen direkt an den EinDollarBrille e.V. Aber keine Panik, das kriegt Ihr locker hin! <3",
+    "DESCRIPTION_4": "Das heißt die Gesamtkosten pro Team betragen 60,-€ (20€ Anmeldegebühr pro Person + 20€ Spendendeposit). Für eine erfolreiche Anmeldung muss der Gesamtbetrag bis zum 16. Mai 2018 überwiesen worden sein. Nutzt hierfür den <strong>angegebenen Verwendungszweck</strong> und folgende Kontoverbindung:",
```
Comment (thepiwo) [inline src/common/resources/translations/translations.de.js:308]: Is the amount the same for 2018?

**Comment by thepiwo** [comment]
what about:
https://github.com/BreakOutEvent/breakout-frontend/blob/3a56a45cb4c579a0fed57f6f76a8758c378daf00/src/server/views/static/content/newsletter.handlebars#L12
https://github.com/BreakOutEvent/breakout-frontend/blob/3a56a45cb4c579a0fed57f6f76a8758c378daf00/src/server/views/layouts/funnel.handlebars#L5
https://github.com/BreakOutEvent/breakout-frontend/blob/3a56a45cb4c579a0fed57f6f76a8758c378daf00/src/server/views/layouts/funnel.handlebars#L24

**Review by thepiwo — COMMENTED** [review]
maybe extract the changes in variables, so they can be adjusted at a central location each year?

**Comment by florianschmidt1994** [comment]
I just created this: https://github.com/BreakOutEvent/breakout-frontend/issues/288

**Review by thepiwo — APPROVED** [review]

File: src/common/resources/translations/translations.de.js (hunk 1 above)
Comment (florianschmidt1994) [reply src/common/resources/translations/translations.de.js:308]: Yes

**Pull request merged**